builtin/providers/postgresql/provider.go

```diff
@@ -11,62 +14,93 @@ func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
 			"host": {
-				Type:        schema.TypeString,
-				Required:    true,
+				Type:     schema.TypeString,
+				Optional: true,
+				// TODO(sean@): Remove POSTGRESQL_HOST in 0.8
```

Comment:
I'd be okay with that.  I've had this kicking around in some form since the mid-0.7 days.  Removing now.